checks/batch/__init__.py

```diff
@@ -595,4 +630,47 @@
             }
         ]
     },
+    {
+        'name': 'mail_rpki',
+        'type': 'category',
+        'translation_key': 'siterpki',
+        'children': [
+            {
+                'name': 'mail_rpki_exists',
+                'name_on_report': 'mail_rpki_exists',
+                'type': 'test',
+                'translation_key': 'mail rpki exists',
+            },
+            {
+                'name': 'mail_rpki_valid',
+                'name_on_report': 'mail_rpki_valid',
+                'type': 'test',
+                'translation_key': 'mail rpki valid',
+            },
+            {
+                'name': 'ns_rpki_exists',
+                'name_on_report': 'ns_rpki_exists',
+                'type': 'test',
+                'translation_key': 'web-mail rpki ns-exists',
+            },
+            {
+                'name': 'ns_rpki_valid',
```

Comment:
'mail_ns_rpki_valid'
```suggestion
                'name': 'mail_ns_rpki_valid',
```